src/server/index.ts

```diff
@@ -0,0 +1,16 @@
+﻿import { createServer } from "http";
+
+import express from "express";
+import helmet from "helmet";
+
+import routes from "./routes";
+
+
+const app = express();
+const server = createServer(app);
+
+app.use(helmet());
```

Comment:
Please take a look at this. By default, it's including **every** middleware to mitigate, which causes issues in environments like development. For example, I can't use the site since it tries to use HTTPS when it's running in dev, forcing me to setup a reverse proxy beforehand.

Reply:
Would it be fine if I added `if (process.env.NODE_ENV === "production")` in front of it?

Reply:
If you do something to ensure `NODE_ENV` is defined, then yes. Presumably, something like `if (process.env.NODE_ENV !== 'development')` would fit better, and then use `cross-env` to define `NODE_ENV=development` in `yarn dev`.